lib/defql/macros/defselect.ex

```diff
@@ -4,27 +4,29 @@ defmodule Defql.Macros.Defselect do
   alias Defql.Connection
 
   @doc false
-  defmacro defselect({name, _, params}, [table: table]) when is_atom(name) and is_list(params) and length(params) == 1 and is_atom(table) do
+  defmacro defselect(_name_params, _opts \\ [])
+  @doc false
+  defmacro defselect({name, _, params}, [columns: columns] = opts) when is_atom(name) and is_list(params) and length(params) == 1 and is_list(columns) do
```

Comment:
here is the method with `columns` param on the first place, because it's head is more certain than `defmacro defselect({name, _, params}, opts)`